### ISSUE TITLE
Campaign expiration stack trace when invalid date is highlighted

### ISSUE THREAD

**Issue by Gilks** [securestate/king-phisher#265]
## Issue Description
On the campaign expiration screen there is an interesting bug. If a date is highlighted that does not exist in another month a stack trace occurs.

For example, if you select March 29th and then attempt to press the back arrow to select a date in February a stack trace will occur because February 29th does not exist.

## Reproduction Steps
1. Create a new campaign
2. On the campaign expiration screen, click the date "March 29th"
3. Press the back arrow to change the month from March to February

## Environment Details

``Linux base 4.10.0-42-generic #46-Ubuntu SMP 2017 x86_64 x86_64 x86_64 GNU/Linux``
``KingPhisher Version: 1.10.0-beta1 (rev: 38a40000736c)``

## Error Details / Stack Trace

```
Error Type: builtins.ValueError
Error Details: ValueError('day is out of range for month',)
Error UID: 8fac8ccc-bfbe-446e-bc29-037cc354370f
RPC Error: N/A (Not a remote RPC error)
King Phisher Version: 1.10.0-beta1 (rev: 38a40000736c)
Platform Version: Linux: Ubuntu 17.04 zesty
Python Version: 3.5.3
Gtk Version: 3.22.11
Timezone: America/New_York

Thread Information:
 => MainThread (alive=True daemon=False)
    Thread-7 (alive=True daemon=True)
    Thread-11 (alive=True daemon=True)
    Thread-6 (alive=True daemon=False)
    Thread-12 (alive=True daemon=True)

Stack Trace:
Traceback (most recent call last):
  File "/opt/king-phisher/king_phisher/client/assistants/campaign.py", line 373, in signal_calendar_prev
    calendar_day = gui_utilities.gtk_calendar_get_pydate(calendar)
  File "/opt/king-phisher/king_phisher/client/gui_utilities.py", line 224, in gtk_calendar_get_pydate
    return datetime.date(calendar_day[0], calendar_day[1] + 1, calendar_day[2])
ValueError: day is out of range for month

```

**Comment by Gilks** [comment]
I've updated to ``KingPhisher Version: 1.10.0-beta2 (rev: cd099361e912)``. I am still able to reproduce this bug on the latest version.

**Comment by zeroSteiner** [comment]
Cool thanks for that information.

**Comment by zeroSteiner** [comment]
Fixed in commit 3a897746be934d78e22bee9557f165f400a4c3aa.
